C/services/common/service_security.cpp

```diff
@@ -0,0 +1,380 @@
+#include <config_category.h>
+#include <string>
+#include <management_client.h>
+#include <service_handler.h>
+#include <config_handler.h>
+#include <server_http.hpp>
+
+#define TO_STRING(...) DEFER(TO_STRING_)(__VA_ARGS__)
```

Comment:
We have various copies of these macros. We really should make them appear in one place someday.